config/forcedPhotCcd.py

```diff
@@ -2,8 +2,10 @@
 
 root.measurement.load(os.path.join(os.environ['OBS_SUBARU_DIR'], 'config', 'apertures.py'))
 root.measurement.load(os.path.join(os.environ['OBS_SUBARU_DIR'], 'config', 'kron.py'))
-root.measurement.load(os.path.join(os.environ['OBS_SUBARU_DIR'], 'config', 'cmodel.py'))
+# Turn off cmodel until latest fixes (large blends, footprint merging, etc.) are in
+# root.measurement.load(os.path.join(os.environ['OBS_SUBARU_DIR'], 'config', 'cmodel.py'))
 
 # Disable Gaussian mags (which aren't really forced)
 root.measurement.algorithms.names -= ["base_GaussianFlux"]
+root.measurement.algorithms.names |= ["base_SdssShape"]
```

Comment:
Oh, I added that to have more measurements to compare with in the forced vs. meas catalogs.  I'll remove it and the one above in both files.